packages/framework/assets/js/admin/validation/form/validationPriceList.js

```diff
@@ -0,0 +1,16 @@
+import Register from '../../../common/utils/Register';
+
+export default function priceListValidator ($container) {
+    window.$('form[name="price_list_form"]').jsFormValidator({
```

Comment:
```suggestion
    $container.filterAllNodes('form[name="price_list_form"]').jsFormValidator({
```
To prevent multiple registrations